README.md

```diff
@@ -134,6 +134,39 @@ Jobs reported to Sauce Labs:
   └────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
 
+## Upload Assets Task
+
+This task allows you to upload assets (such as images or logs) to a specific Sauce Labs job associated with the test spec.
+
+| Parameter           | Type     | Description                                                                                                                         |
+| ------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `spec`              | `string` | Path to the spec file being executed, typically provided by `__filename`.                                                           |
+| `assets`            | `array`  | Array of asset objects to be uploaded to Sauce Labs, each containing a `filename` and either `path` or `data`.                      |
+| `assets[].filename` | `string` | **Required.** The name of the file to upload, as it should appear in Sauce Labs (e.g., `"this-is-fine.png"`).                       |
+| `assets[].path`     | `string` | **Optional.** Path to the file on the local filesystem (e.g., `"pics/this-is-fine.png"`). Either `path` or `data` must be provided. |
+| `assets[].data`     | `stream` | **Optional.** File data as a stream, used when directly providing file content. Either `path` or `data` must be provided.           |
+
+### Example Usage
+
+```javascript
+it("upload assets", () => {
+  // Single file upload.
+  cy.task("sauce:uploadAssets", {
```

Comment:
We have an integration test in CI. Would be good if it upload some file there as well 🙏

Reply:
👍 Good idea!